evm/test/forks/HubTestConstants.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity ^0.8.23;
+
+import {HubGovernor} from "src/HubGovernor.sol";
+import {HubVotePool} from "src/HubVotePool.sol";
+
+contract HubTestConstants {
+  // Deployed Contract Addresses (from mainnet-test-deploy-contracts.md)
+  address payable constant GOV_ADDR = payable(0x239B1F17E6Efa75662cB87781025538babF1Cf6b);
+  address immutable TIMELOCK_ADDR = 0xfBc580c0289121673EfB7375fF111bD2A4db4654;
+  address immutable EXTENDER_ADDR = 0x3dDeaA121C33eDBF802984096aB450a6051E0e73;
+  address immutable HUB_VOTE_POOL_ADDR = 0x2E57935d31Ef7F161e7bC69Fca873E04097ff3af;
+  address immutable HUB_EVM_VOTE_DECODER_ADDR = 0x210dD4E9F8da7967792b0Ff73a340140aF6A2115;
+  address immutable HUB_SOLANA_VOTE_DECODER_ADDR = 0xACA7A131B94E6972B9BB7A48Ff6f5f1E8181Fa1E;
+  address constant HUB_METADATA_ADDR = 0xE92610200A579A09Fe440f697C19afadDa8a7fE8;
+  address constant HUB_MSG_DISPATCHER_ADDR = 0x9B3679B7e3E51d4f1E0eeF3977B400011365CbCE;
+  address constant HUB_SOLANA_DISPATCHER_ADDR = 0x6b608339E64662F49425b02e0c82ea904492BFab;
+  address constant HUB_EVM_AGG_PROPOSER_ADDR = 0xC7D279Fcd5b122BDEc63112F31967b2eAB080Af6;
+  // // Testnet WToken address (replace with actual for prod verification)
```

Comment:
```suggestion
```

should remove now that it's updated with real w token

Reply:
https://github.com/wormhole-foundation/multigov/pull/269/commits/b485b8a6800a50672361eafcd87173e495c200d3